src/libraries/System.Collections.Immutable/tests/default.rd.xml

```diff
@@ -37,24 +37,108 @@
         </Method>
       </Type>
 
+      <Type Name="Xunit.Sdk.AssertEqualityComparer`1[[System.Collections.Generic.HashSet`1[[System.Collections.Generic.KeyValuePair`2[[System.SByte, System.Private.CoreLib],[System.SByte, System.Private.CoreLib]],System.Private.CoreLib]],System.Private.CoreLib]]" Dynamic="Required All">
```

Comment:
@agocke, do you have a timeline when we will have Xunit asserts that work in NativeAOT?

Reply:
I have it in a branch locally, and just keep getting interrupted by other stuff. I will likely work on it more after we shut down the platform, as infra will be more important then.